plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1877,6 +1877,11 @@ dialog.workspace.is_not_valid_message=<html><big>Workspace open failed!</big><br
 dialog.workspace.is_not_valid_title=Workspace open failed
 dialog.workspace.unknown_generator_message=<html><b>This workspace uses unsupported generator type: {0}</b><br>Select one of the supported generators for the workspace.
 dialog.workspace.unknown_generator_title=Unsupported generator
+dialog.workspace.version_locked_message=<html><b>Workspace is locked to a different version.</b><br><br>\
```

Comment:
Workspace version is an ambiguous term. Is it the workspace version of the workspace itself, or the MCreator version? This should be expressed more clearly as "workspace MCreator version" or similar.

I would also include version number it is locked to in this message so the user can easily know what version exactly they locked to.